### PR TITLE
lf: install manpage and shell completion

### PR DESCRIPTION
**Pull request by thezeroalpha**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/lf.rb [hunk 1]
```diff
@@ -20,6 +21,9 @@ def install
     cd "src/github.com/gokcehan/lf" do
       system "./gen/build.sh", "-o", bin/"lf"
       prefix.install_metafiles
+      man1.install "lf.1"
+      zsh_completion.install "etc/lf.zsh" => "_lf"
+      fish_completion.install "etc/lf.fish"
```
Comment (SMillerDev) [inline Formula/lf.rb:26]: Is there no bash completion?

Comment (thezeroalpha) [reply Formula/lf.rb:26]: No, the authors have not written a bash completion file yet.

**Comment by SMillerDev** [comment]
Thanks @thezeroalpha ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by thezeroalpha** [comment]
Thanks for your help on the forum @SMillerDev!

File: Formula/lf.rb (hunk 1 above)
Comment (issyl0) [reply Formula/lf.rb:26]: It's nice to see this the other way around for once - it's usually bash that comes first!

**Pull request closed without merging**